### PR TITLE
Iep 621 update response bodies for lresex

### PR DESCRIPTION
**Pull request by CHssmith2**

### CODE REVIEW

File: transformers/resolution_resource.go [hunk 1]
```diff
@@ -1,14 +1,31 @@
...
+
+	"github.com/companieshouse/chs.go/log"
 	"github.com/companieshouse/insolvency-api/models"
+	"github.com/companieshouse/insolvency-api/utils"
 )
 
 // ResolutionResourceRequestToDB transforms a resolution request to a dao model
-func ResolutionResourceRequestToDB(req *models.Resolution) *models.ResolutionResourceDao {
+func ResolutionResourceRequestToDB(req *models.Resolution, transactionID string) *models.ResolutionResourceDao {
+	etag, err := utils.GenerateEtag()
```
Comment (Jideboris) [inline transformers/resolution_resource.go:13]: There is a need for a check on etag before continue its value is needed in following lines.

File: transformers/resolution_resource.go [hunk 2]
```diff
@@ -1,14 +1,31 @@
...
+	if err != nil {
```
Comment (Jideboris) [inline transformers/resolution_resource.go:14]: The code should return here with error logs for etag is not generated.

File: transformers/resolution_resource.go [hunk 3]
```diff
@@ -1,14 +1,31 @@
...
+		log.Error(fmt.Errorf("error generating etag: [%s]", err))
+	}
+
+	kind := "insolvency-resource#resolution"
```
Comment (Jideboris) [inline transformers/resolution_resource.go:18]: I do not think we need this declaration for kind.We could pass the string value directly into the struct.

File: transformers/resolution_resource.go (hunk 2 above)
Comment (Jideboris) [reply transformers/resolution_resource.go:15]: Fixed

File: transformers/resolution_resource.go (hunk 3 above)
Comment (Jideboris) [reply transformers/resolution_resource.go]: Fixed

**Comment by Jideboris** [comment]
I have got the branch down locally, just waiting on increasing its test coverage.

**Review by adeharrison-ch — APPROVED** [review]
get and post look good to me

**Pull request merged**